lib/agenda.js

```diff
@@ -36,7 +38,52 @@ WHERE {
   return null;
 }
 
+async function getAgendasForSubcase(subcaseId, useSudo=false) {
+  // "prov:wasRevisionOf" predicate itself can be propagated already even if the newer agenda was not
+  // for other graphs we need to verify if that newer agenda exists or not f.e. by checking id
+  const queryString = `PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX besluitvorming: <https://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+
+SELECT DISTINCT (?meeting AS ?uri) ?meetingId ?agendaId ?agendaitemId ?plannedStart ?status ?kind ?number
+${useSudo ? `FROM ${sparqlEscapeUri(GRAPHS.KANSELARIJ)}` : ''}
+WHERE {
+  ?subcase a dossier:Procedurestap ;
+    mu:uuid ${sparqlEscapeString(subcaseId)} ;
+    ^besluitvorming:vindtPlaatsTijdens ?agendaActivity .
+
+  ?agendaActivity besluitvorming:genereertAgendapunt ?agendaitem .
+  
+  ?agendaitem mu:uuid ?agendaitemId .
+
+  ?agenda dct:hasPart ?agendaitem ;
+    mu:uuid ?agendaId ;
+    besluitvorming:isAgendaVoor ?meeting ;
+    besluitvorming:agendaStatus ?status .
+  FILTER NOT EXISTS { ?newerId ^mu:uuid/prov:wasRevisionOf ?agenda }
+
+  ?meeting mu:uuid ?meetingId ;
+    dct:type ?kind ;
+    adms:identifier ?number ;
```

Comment:
adms:identifier seems optional. Checked on PROD data with
 ```
  PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
 PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
 PREFIX besluitvorming: <https://data.vlaanderen.be/ns/besluitvorming#>
 PREFIX dct: <http://purl.org/dc/terms/>
 PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
 PREFIX prov: <http://www.w3.org/ns/prov#>
 PREFIX adms: <http://www.w3.org/ns/adms#>
 
 SELECT ?plannedStart
 
 WHERE { 
GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
   ?meeting mu:uuid ?meetingId ;
     besluit:geplandeStart ?plannedStart .
FILTER NOT EXISTS {  ?meeting  adms:identifier ?number .}

 }}
 ORDER BY DESC(?plannedStart)
 ```
 
 60 meetings that have no identifier

Reply:
Fixed that.
The only downside is=
no number in the view (locally)
![image](https://github.com/user-attachments/assets/5fcf84b3-a81a-4c83-bc36-283ba32aac48)

But looking at PROD data. this is the same now:
![image](https://github.com/user-attachments/assets/77c3de03-f23c-4f5e-a4f1-8798479e8b03)